genesyscloud/webdeployments_deployment/genesyscloud_webdeployments_deployment_proxy.go

```diff
@@ -85,3 +97,46 @@ func updateWebdeploymentsFn(ctx context.Context, p *webDeploymentsProxy, deploym
 func deleteWebdeploymentsFn(ctx context.Context, p *webDeploymentsProxy, deploymentId string) (*platformclientv2.APIResponse, error) {
 	return p.webDeploymentsApi.DeleteWebdeploymentsDeployment(deploymentId)
 }
+
+func determineLatestVersionFn(ctx context.Context, p *webDeploymentsProxy, configurationId string) (string, []string) {
+	version := ""
+	draft := "DRAFT"
+	versionList := []string{}
+	_ = util.WithRetries(ctx, 30*time.Second, func() *retry.RetryError {
```

Comment:
Shouldn't we be catching any errors here, currently if there's an error it will be lost right?